e2e/healthcheck_test.go

```diff
@@ -4,23 +4,28 @@ import (
 	"net/http"
 	"os"
 	"os/exec"
-	"time"
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/gbytes"
 	"github.com/onsi/gomega/gexec"
 )
 
+const (
+	// kFlag is a keyword to check whether the healthcheck is ready (or process is died)
+	kFlag = "kflag"
```

Comment:
```golang
finalizerKeyWord = "exiting"
```

Reply:
Actually, we could have the other wording better than finalizer because it includes the case that start listening too.

Reply:
Start listening?

We should only print it when it fails with a fatal error, so that kepler will not be running

Reply:
This is true and that's why the health checking exists!

So we don't need the string to say kepler failed, but to say it has started... 
And then the test will try the health check to see if the kepler is still alive...

What we want then is a log that prints after the server is started

Anyway, as we discussed before let's correct the logic in another PR....